### PR TITLE
Upgrade Ogmios to latest master

### PR DESCRIPTION
**Pull request by ngua**
Adds `aarch64-{linux|darwin}` support for Ogmios

### CODE REVIEW

**Review by firefrorefiddle — APPROVED** [review]
Looks good!

File: flake.nix [hunk 1]
```diff
@@ -8,7 +8,7 @@
     };
 
     # for the purescript project
-    ogmios.url = "github:mlabs-haskell/ogmios/c4f896bf32ad066be8edd8681ee11e4ab059be7f";
+    ogmios.url = "github:mlabs-haskell/ogmios";
```
Comment (klntsky) [inline flake.nix:11]: Why did you add revision here and not to `locked.rev`?

Comment (ngua) [reply flake.nix:11]: I had to specify a specific commit previously as I hadn't merged the changes into Ogmios master yet. You can also do that with the `rev` attribute, but that requires the long form:
```
ogmios = {
  type = "github";
  owner = "...";
  repo = "...";
  rev = "...";
};
```

`locked.rev` is in the lockfile, which you shouldn't edit by hand
